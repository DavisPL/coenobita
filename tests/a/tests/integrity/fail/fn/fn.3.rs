#![allow(unused_variables)]
#![allow(unused_assignments)]

#[cnbt::integrity({*}{*} fn({a,root}{a,root}) -> {a,root}{a,root})]
fn id(x: i32) -> i32 {
    x
}

fn main() {
    for i in 0..c::value() {
        let _ = id(5);
    }
}