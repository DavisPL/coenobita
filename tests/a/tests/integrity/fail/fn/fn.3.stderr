error: expected {a,bin}{a,bin}, found {bin}{bin,c}
  --> tests/integrity/fail/fn/fn.3.rs:11:20
   |
11 |         let _ = id(5);
   |                    ^
